src/js/api/order.js

```diff
@@ -2052,6 +2052,56 @@ ripe.Ripe.prototype.setOrderStatus = function(number, status, options, callback)
     return this._cacheURL(options.url, options, callback);
 };
 
+/**
+ * Sets the Order size in its details.
+ *
+ * @param {Number} number The number of the order to set the size.
+ * @param {string} size The size.

```

Comment:
Missing @param options and @param callback, check other functions jsdocs.

Reply:
There is not a description for the callback param to be re-used, but since it is so common, perhaps it would not be necessary to put a description for it, nevertheless I left a line.

Reply:
https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/api/order.js#L17

Reply:
Sorry, "missearched" it. code ammended.